test/config_test.js

```diff
@@ -1,58 +1,153 @@
 suite("config", function() {
-  var assert  = require('assert');
-  var config = require('../lib/config');
+  var config  = require('../');
```

Comment:
I think this is a sane way to load, right?

Ie. we'll get the compiled version... :)